index.html

```diff
@@ -129,7 +130,12 @@ <h2><span class="red">Profits</span> Over People.</h2>
 			</section>
 			<section id="join" class="container">
 				<div class="wrapper">
-					<h2 class="action-head">Take <span class="red">Action</span>:</h2>
+					<h2 class="action-head" id="take-action">Take <span class="red">Action</span>:
+						<div class="header-copy"><noscript><a href="#join"></a></noscript></div>
+					</h2>
+					<script>
+						addCopyButton(document.getElementById("take-action"), location.protocol + '//' + location.host + location.pathname + "#join")
+					</script>
```

Comment:
Is there a reason for this instead of using the header-link script?

Reply:
Yes, my thinking was that it would link to `#join` instead of `#take-action`. That can be changed if we want though.

Reply:
Ah, thanks, makes sense to me!